verif/cache/cache_pp.py

```diff
@@ -99,9 +99,11 @@ def print_message(msg):
     else:
         print_message(f"\n[ERROR] {args.test_name} have failed Post-Process")
         sys.exit(1)
-else: 
-    print_message(f"\n[WARNING] No golden tracker found for test {args.test_name}")
-    sys.exit(0)
+
+#No need anymore because of level0
```

Comment:
Lets remember to remove this.
Not only because of the level - we have the ref modek that generates logs for every tests